app/javascript/components/bulk_edit/CaseStatus.js

```diff
@@ -0,0 +1,237 @@
+import React from 'react';
+import { Button, Modal, Form } from 'react-bootstrap';
+import { PropTypes } from 'prop-types';
+import axios from 'axios';
+import reportError from '../util/ReportError';
+
+class CaseStatus extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showModal: false,
+      patients: [],
+      case_status: '',
+      message: '',
+      confirmed: '',
+      isolation: '',
+      apply_to_group: false,
+      monitoring: false,
+      monitoring_reason: '',
+      public_health_action: '',
+      loading: false,
+    };
+    this.submit = this.submit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.toggleCaseStatusModal = this.toggleCaseStatusModal.bind(this);
+
+    window.bulkEditCaseStatusComponent = this;
+  }
+
+  activate(patients) {
+    if (!patients || !patients.length) {
+      return;
+    }
+
+    this.getCommonAttributes(patients);
+
+    this.setState({
+      showModal: true,
+      patients: patients,
+    });
+  }
+
+  toggleCaseStatusModal() {
+    let current = this.state.showModal;
+    this.setState({
+      showModal: !current,
+    });
+  }
+
+  // Creates a dictionary, tracking the count of common attributes. This list is used to pre-populate
+  // the form fields and can be displayed to the user for knowledge of the selection.
+  getCommonAttributes(patients) {
+    if (patients.length == 0) return '';
```

Comment:
@holmesie is eslint not enforcing things like this? https://eslint.org/docs/rules/eqeqeq

Reply:
It is, but in the static analysis CI run, brakeman gets called first (and the moment something fails the whole run stops). Brakeman is complaining about something, so the linter didn't get run. Not sure why this wasn't flagged locally though, will need to look into that.